app/src/main/java/com/keylesspalace/tusky/view/ExposedPlayPauseVideoView.kt

```diff
@@ -12,19 +12,26 @@ class ExposedPlayPauseVideoView @JvmOverloads constructor(
     VideoView(context, attrs, defStyleAttr) {
 
     private var listener: PlayPauseListener? = null
+    private var playing = false
 
     fun setPlayPauseListener(listener: PlayPauseListener) {
         this.listener = listener
     }
 
     override fun start() {
         super.start()
-        listener?.onPlay()
+        if (!playing) {
```

Comment:
start is called again when the surface size changes. Added the check to not trigger a double-call to the listener.